packages/gatsby/index.d.ts

```diff
@@ -1529,9 +1529,9 @@ export interface PluginOptionsSchemaArgs {
 type Send<T> = (body: T) => void
 
 /**
- * Gatsby API Function route response
+ * Gatsby Function route response
  */
-export interface GatsbyAPIFunctionResponse extends ServerResponse {
+export interface GatsbyFunctionResponse extends ServerResponse {
```

Comment:
@vladar this will break TS in Gatsby, see https://github.com/gatsbyjs/gatsby/pull/31182

Reply:
I guess we should backport that PR too? And curious why it is not caught by tests.